activesupport/test/core_ext/object/exclusion_test.rb

```diff
@@ -0,0 +1,54 @@
+require 'abstract_unit'
+require 'active_support/core_ext/object/exclusion'
+
+class NotInTest < ActiveSupport::TestCase
+  def test_not_in_array
+    assert 1.not_in?([2, 3])
+    assert !2.not_in?([1,2])
```

Comment:
I copied the tests verbatim from `inclusion_test.rb`. Want me to change those? Or keep this atomic?